internal/stack/resources.go

```diff
@@ -0,0 +1,163 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package stack
+
+import (
+	"embed"
+	"fmt"
+	"html/template"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/elastic/go-resource"
+
+	"github.com/elastic/elastic-package/internal/profile"
+)
+
+//go:embed _static
+var static embed.FS
+
+const (
+	// SnapshotFile is the docker-compose snapshot.yml file name.
+	SnapshotFile = "snapshot.yml"
+
+	// ElasticsearchConfigFile is the elasticsearch config file.
+	ElasticsearchConfigFile = "elasticsearch.yml"
+
+	// KibanaConfigFile is the kibana config file.
+	KibanaConfigFile = "kibana.yml"
+
+	// KibanaHealthcheckFile is the kibana healthcheck.
+	KibanaHealthcheckFile = "kibana_healthcheck.sh"
+
+	// PackageRegistryConfigFile is the config file for the Elastic Package registry
+	PackageRegistryConfigFile = "package-registry.yml"
+
+	// PackageRegistryBaseImage is the base Docker image of the Elastic Package Registry.
+	PackageRegistryBaseImage = "docker.elastic.co/package-registry/package-registry:v1.18.0"
```

Comment:
A new version of package-registry has been merged into elastic-package `v1.19.0`

Reply:
Updated, thanks!